src/lib/webpack/prerender.js

```diff
@@ -1,17 +1,17 @@
+import chalk from 'chalk';
 import { resolve } from 'path';
-import fs from 'fs';
+import { readFileSync } from 'fs';
 import stackTrace from 'stack-trace';
 import { SourceMapConsumer } from 'source-map';
-import chalk from 'chalk';
 
 export default function prerender(env, params) {
 	params = params || {};
 
-	let entry = resolve(env.dest, './ssr-build/ssr-bundle.js'),
-		url = params.url || '/';
+	let entry = resolve(env.dest, './ssr-build/ssr-bundle.js');
+	let url = params.url || '/';
```

Comment:
is `let` neccesary here? Can this be a `const`?

Reply:
@developit prefers `let` for most things. I broke the rule & used `const` a few times to be emphatic.